### PR TITLE
Update SFN -> Lambda merging instructions

### PR DESCRIPTION
**Pull request by avedmala**
### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/serverless/step_functions/merge-step-functions-lambda.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ For Python: Datadog Lambda Library for Python layer 99+ **or** `dd-trace-py` v2.
 
 If the layer or tracer version requirements are fulfilled, no further setup is required.
 
+<div class="alert alert-info">Ensure that some input, even if empty, is provided to the Step Functions Start Execution command for trace merging to function correctly.</div>
```
Comment (neko-dd) [inline content/en/serverless/step_functions/merge-step-functions-lambda.md:114]:
```suggestion
<div class="alert alert-info">To ensure that trace merging functions correctly, provide input to the Step Functions Start Execution command. The input can be empty.</div>
```

Is this referring to an empty string? What does empty input mean?

Comment (avedmala) [reply content/en/serverless/step_functions/merge-step-functions-lambda.md]: It's referring to stringified JSON input here

Comment (neko-dd) [reply content/en/serverless/step_functions/merge-step-functions-lambda.md]: Okay. Hopefully this is obvious enough.

**Review by neko-dd — APPROVED** [review]

File: content/en/serverless/step_functions/merge-step-functions-lambda.md (hunk 1 above)
Comment (neko-dd) [reply content/en/serverless/step_functions/merge-step-functions-lambda.md]: Ah, I see you changed my suggestion to be more specific. Excellent! Thanks.

**Pull request merged**